js/components/Header.js

```diff
@@ -74,7 +74,7 @@ export default class Header extends Component {
 Header.propTypes = {
   title: PropTypes.string,
   titleColor: PropTypes.string,
-  style: PropTypes.object,
+  style: View.propTypes.style,
```

Comment:
It basically means you can pass style as either an object (`{ flex: 1 }`) or a `Stylesheet` property (`styles.foo`). It extends how RN views handle style.